setup.py

```diff
@@ -29,7 +29,7 @@
 
 setup(
     name = 'ifaddr',
-    version = '0.1.6',
+    version = '0.1.7',
```

Comment:
Please leave the version intact, I'll bump it at a later point in a separate commit/PR.

Reply:
👍